core/src/main/java/fj/data/List.java

```diff
@@ -1035,7 +1084,7 @@ public final boolean exists(final F<A, Boolean> f) {
    * @return a list of all the items in this list that do not appear in the given list.
    */
   public final List<A> minus(final Equal<A> eq, final List<A> xs) {
-    return removeAll(compose(Monoid.disjunctionMonoid.sumLeft(), xs.mapM(curry(eq.eq()))));
+    return this.removeAll(compose(Monoid.disjunctionMonoid.sumLeft(), xs.mapM(Function.<A, A, Boolean>curry(eq.eq()))));
   }
```

Comment:
to make Intellij happy :). There are many places Intellij fails to infer types. not sure if we want to make Intellij happy?